section8_function_inputs/project/main.py

```diff
@@ -0,0 +1,66 @@
+alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+
+def encrypt(message, shifting, alphabet_length = len(alphabet)):
+    caesar_coding = ""
+    # Looping every letter in text.
+    for each_letter in range(len(message)): 
+        encrypt_letter = message[each_letter]
+        # If the letter is in alphabet list, encrypt program is going to encode each letter.
+        if message[each_letter] in alphabet:
```

Comment:
You should use encrypt_letter instead of message[each_letter] since you already assigned it to encrypt_letter